BUGFIX: Update inline hidden state when hidden property is changed via inspector

Sebobo opened the pull request:
This change handles the case that the hidden state is updated in the inspector which didn't update the guest frame.
Also the case is handled that reloadIfChanged is enabled on the hidden property but the ui didn't send a nodeDomAdress and the backend would throw an exception.

Resolves: #3988

Sebobo committed:
BUGFIX: Update inline hidden state when hidden property is changed via inspector

Resolves: #3988

dlubitz reviewed (approved):
Looks good to me! Thank you for taking care.

Sebobo committed:
BUGFIX: Handle case of requested reload with missing nodeDomAddress

This case was handled already in the other branches of the AbstractChange but one case was missing.

mhsdesign reviewed (approved):
Thanks for the fix. ... also not requiring the dom address for the property might make kinda sense ... but we can always remove that later ^^. `properties._hidden.ui.reloadIfChanged` definetely is not a usecase
